### PR TITLE
Localization for dotnet-clean command.

### PR DESCRIPTION
**Pull request by vsccarl**
Added localization for the clean command. All localizable strings from the command are now placed in a new file, LocalizableStrings.cs.

### CODE REVIEW

**Comment by vsccarl** [comment]
@dotnet-bot Test Ubuntu16.04 x64 Debug Build

**Comment by vsccarl** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build

**Comment by vsccarl** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build

**Comment by vsccarl** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build

**Comment by piotrpMSFT** [comment]
@MattGertz

File: src/dotnet/commands/dotnet-clean/Program.cs [hunk 1]
```diff
@@ -17,20 +17,19 @@ public static int Run(string[] args)
...
-            CommandArgument projectArgument = app.Argument("<PROJECT>",
-                "The MSBuild project file to build. If a project file is not specified," +
-                " MSBuild searches the current working directory for a file that has a file extension that ends in `proj` and uses that file.");
+            CommandArgument projectArgument = app.Argument($"<{LocalizableStrings.CmdArgProject}>",
+                LocalizableStrings.CmdArgProjDescription);
 
-            CommandOption outputOption = app.Option("-o|--output <OUTPUT_DIR>", "Directory in which the build outputs have been placed", CommandOptionType.SingleValue);
-            CommandOption frameworkOption = app.Option("-f|--framework <FRAMEWORK>", "Clean a specific framework", CommandOptionType.SingleValue);
-            CommandOption configurationOption = app.Option("-c|--configuration <CONFIGURATION>", "Clean a specific configuration", CommandOptionType.SingleValue);
+            CommandOption outputOption = app.Option($"-o|--output <{LocalizableStrings.CmdOutputDir}>", LocalizableStrings.CmdOutputDirDescription, CommandOptionType.SingleValue);
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-clean/Program.cs]: NIT: please, keep the lines to less than 120 characters.

And when you break the lines, I would prefer that when you put parameters in a new line that each of them have their own new line. Line this:

cmd.Option(
$"...",
Localiza...,
CommandOptionType...);

File: src/dotnet/commands/dotnet-clean/Program.cs [hunk 2]
```diff
@@ -17,20 +17,28 @@ public static int Run(string[] args)
...
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
             app.HelpOption("-h|--help");
 
-            CommandArgument projectArgument = app.Argument("<PROJECT>",
-                "The MSBuild project file to build. If a project file is not specified," +
-                " MSBuild searches the current working directory for a file that has a file extension that ends in `proj` and uses that file.");
+            CommandArgument projectArgument = app.Argument($"<{LocalizableStrings.CmdArgProject}>",
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-clean/Program.cs]: Move parameters to their own lines when you move any to their own line. You left this one and a couple others. Do a pass and see other cases that may apply.

**Pull request merged**